### DISCUSSION TITLE
Initial Development. Your help

### DISCUSSION THREAD

**Discussion by animafps** [aiming-taxonomy/glossary#6] [Ideas]
Hello people
this is a pretty large project in terms of the amount of formatting and structure as well as organization needed for this to work so we need to figure out these things

The [project board](../projects/1) for initial development is stuff that i want to complete first so we don't start adding stuff with a disorganized system to a badly formatted document as well as anything that can can help develop this in accordance to [ridd's proposed development aproach](1)

if anything on the working on or to do columns you can help out with with ideas or feedback
then create a discussion about it and describe what your idea or feedback is and i or once we get some mergers they can help out and get your idea into the real project

There are a couple of starting concepts definition discussions happening so go discuss

For people who want to help out with management or to be acting as a merger please email or dm me 

Plus we need to get some more people to help out with contributing

**Comment by riddbtw** [comment]
@AnimaFPS - i'm happy to help out as a manager.

I would like to implement a way to limit our work in progress (WIP) and make sure it's totally 'done' before moving onto next for each of the aim concept definitions.  Perhaps we can use that [project board](https://github.com/aiming-taxonomy/aiming-taxonomy/projects/2) as a kanban board, and each of the columns would represent a state each definition is in (i.e. discussion -> voting -> branch / pr -> merge).  Then we only allow a few definitions at a time (3-5) before we pull in a new one to be worked.  Seem reasonable?  Just want to avoid concurrently discussing like 20 topics without merging anything.

**Reply by animafps** [reply]
Well discussion shouldn't really be regulated think its good for the start for the initial ones, because discussions should evolve constantly. While past the initial push and rounds of this WIP it should be based on how much has changed with discussion to lead to a proposal and subsequently a vote. While when we move on to the second stage there can be a way to promote the top 3 at the time highly discussed concepts but shouldn't restrict in this stage.

As well i have invited you through github to join the github org which you can get management rights to do stuff like the board(which i have done) and stuff

**Comment by animafps** [comment]
Think initial development might be fully finished just need to convert the current definitions to threads but need to stratigise for people might just create a reddit post as those people love to discuss and easy promotion for this project but need a way to signal things that are good for people to contribute to but might be good to just make another one of these discussion threads and lock it for the next development stage and send people to contributing.md and the project board and stuff
